provider/execution/execution.go

```diff
@@ -92,6 +92,7 @@ func (e TableExecutor) withTable(t *schema.Table, kv ...interface{}) *TableExecu
 	var c [2]schema.ColumnList
 	c[0], c[1] = e.Db.Dialect().Columns(t).Sift()
 	cpy := e
+	cpy.Table.Parent = t.Parent
```

Comment:
You don't need this since you have line 96?

Reply:
I think I do because parent is not set in table automatically (i.e by the user). I can also move this function to execution and not to table if that will make more sense.

Reply:
But this is overridden by L96, so it has no effect?